tests/modules/asset_groups/resources/test_commit_asset_group.py

```diff
@@ -20,8 +20,13 @@ def test_commit_asset_group(flask_app_client, researcher_1, regular_user, test_r
         asset_group_uuid = response.json['guid']
         asset_group_sighting_guid = response.json['sightings'][0]['guid']
 
+        # Should not be able to commit as contributor
+        asset_group_utils.commit_asset_group_sighting(
+            flask_app_client, regular_user, asset_group_sighting_guid, 403
+        )
```

Comment:
Theoretically the UI should prevent the user from seeing an option to do this, but it's good to have the check.

Reply:
I work on the principle that while Bens UI is currently using this IF, If the IF is available for anyone to use, we need to be reasonably robust.